frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -45,8 +45,29 @@ class MenuUser extends StatelessWidget {
     String iconUrl = context.read<MenuUserBloc>().state.userProfile.iconUrl;
     if (iconUrl.isEmpty) {
       iconUrl = defaultUserAvatar;
+      final String name = context.read<MenuUserBloc>().state.userProfile.name;
+      final Color nameColor = _generateRandomNameColor(name);
+      // Taking the first letters of the name components and limiting to 2 elements
+      final nameInitials = name
+          .split(' ')
+          .map((element) => element[0].toUpperCase())
+          .take(2)
+          .join('');
+      return Container(
+        width: 28,
+        height: 28,
+        alignment: Alignment.center,
+        decoration: BoxDecoration(
+          color: nameColor,
+          shape: BoxShape.circle,
+        ),
+        child: FlowyText.semibold(
+          nameInitials,
+          color: Colors.white,
+          fontSize: nameInitials.length == 2 ? 12 : 14,
```

Comment:
Seems a little tricky. It won't work when the username is `aaa bbb cccc ddd`.

Reply:
nameInitials take only the first two values, so we will see 2 initials at maximum

Reply:
Ok. In that case, you can define a const value to indicate the number 2, since you use it twice. It would be better to define it as a value if it's used more than once.

`take(2)` and  `== 2`.